app/notifiers/applicant_will_perform_notifier.rb

```diff
@@ -1,6 +1,25 @@
 # frozen_string_literal: true
 class ApplicantWillPerformNotifier < BaseNotifier
   def self.call(job_user:, owner:)
+    owner_email(job_user: job_user, owner: owner)
+
+    rejected_emails(confirmed_job_user: job_user)
+  end
+
+  def self.rejected_emails(confirmed_job_user:)
+    job_users = confirmed_job_user.job.job_users.includes(:user)
```

Comment:
A bit ugly to fetch the rejected users this way.. but couldn't find an obvious refactor either